serializers/src/glTF/2.0/babylon.glTFExporter.ts

```diff
@@ -1075,62 +1052,68 @@ module BABYLON.GLTF2 {
          * @param babylonScene Babylon scene to get the mesh data from
          * @param binaryWriter Buffer to write binary data to
          */
-        private createScene(babylonScene: Scene, binaryWriter: _BinaryWriter): void {
+        private createSceneAsync(babylonScene: Scene, binaryWriter: _BinaryWriter): Promise<void> {
             if (this.setNodeTransformation.length) {

```

Comment:
I don't understand this line. What are you trying to check?

Reply:
I don't know why I used that line there either...  I'm taking it out